public/index.php

```diff
@@ -1,12 +1,27 @@
 <?php
 
+use App\Controllers\UserManagementController;
 use App\Kernel;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
+$_GET['id'] = 1;
```

Comment:
$_GET['id'] = 1; jest niepotrzebne ponieważ ID powinno się podawać w adresie URL
http://localhost/public/index.php?controller=usermanagement&do=show&id=1

W wypadku gdy chcę załadować w ten sposób użytkownika o ID 2 to program nie działa ponieważ ładuje cały czas użytkownika numer 1.

Jeżeli id w parametrze URL nie jest podane wtedy prawidłowo program wyrzuca błąd.